src/style.scss

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form {
+  &__input {
+    box-sizing: border-box;
+    display: flex;
+    width: 100%;
+
+    margin-top: 20px;
+    padding: 0;
+    font-family: Avenir, Arial, sans-serif;
```

Comment:
According to the checklist, you should use fallback fonts - alternative font-family in case the main one doesn't work. So, in addition to `Avenir` and `Arial`, you should also include a generic font family like `sans-serif`. This is a code quality issue.